src/code/z_fbdemo_triforce.c

```diff
@@ -118,14 +124,13 @@ void TransitionTriforce_Draw(void* thisx, Gfx** gfxP) {
 
 s32 TransitionTriforce_IsDone(void* thisx) {
     TransitionTriforce* this = (TransitionTriforce*)thisx;
-
-    s32 ret = 0;
+    s32 isDone = false;
 
     if (this->state == 1 || this->state == 2) {
         return this->transPos <= 0.03f;
-
     } else if (this->state == 3 || this->state == 4) {
         return this->transPos >= 1.0f;
     }
-    return ret;
+
+    return isDone;
```

Comment:
this matches too, with or without the else
```c
s32 TransitionTriforce_IsDone(void* thisx) {
    TransitionTriforce* this = (TransitionTriforce*)thisx;

    if (this->state == 1 || this->state == 2) {
        return this->transPos <= 0.03f;
    } else if (this->state == 3 || this->state == 4) {
        return this->transPos >= 1.0f;
    } else {
        return false;
    }
}
```
(I think with the else looks nicer)